github/downloader_test.go

```diff
@@ -1,61 +1,62 @@
+// Integration tests of the metadata graphql crawler:
+// - with and without the DB
+// - online and offline tests
+
 package github
 
 import (
+	"bytes"
+	"compress/gzip"
 	"context"
+	"database/sql"
+	"encoding/gob"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"net/http"
 	"os"
+	"runtime"
+	"strings"
 	"testing"
+	"time"
 
+	"github.com/cenkalti/backoff"
+	"github.com/src-d/metadata-retrieval/database"
+	"github.com/src-d/metadata-retrieval/github/store"
 	"github.com/src-d/metadata-retrieval/testutils"
```

Comment:
sorry for pointing at this each time, but these are internal imports :(

Reply:
Yeah sorry. Visual studio code is rearranging stuff all the time and then I forget to manually rearrange them. I should add some pre-commit hook to handle this stuff. Do you have something to propose?

Reply:
I do it manually tbh, but probably there should be some linter for this purpose